sites/fast-website/package.json

```diff
@@ -0,0 +1,40 @@
+{
+  "name": "fast-website",
+  "version": "1.0.0",
+  "description": "fast website",
+  "main": "index.js",
+  "private": true,
+  "license": "MIT",
+  "devDependencies": {
+    "@microsoft/fast-element": "^0.10.2",
+    "@microsoft/fast-foundation": "^1.1.2",
+    "@microsoft/fast-web-utilities": "^4.5.1",
+    "@microsoft/fast-components": "^1.1.2",
+    "babel-core": "^6.26.3",
+    "babel-loader": "^8.1.0",
+    "babel-preset-env": "^1.7.0",
+    "clean-webpack-plugin": "^3.0.0",
+    "html-webpack-plugin": "^4.3.0",
+    "mini-css-extract-plugin": "^0.9.0",
+    "optimize-css-assets-webpack-plugin": "^5.0.3",
+    "prettier": "2.0.2",
+    "svg-inline-loader": "^0.8.2",
+    "terser-webpack-plugin": "^3.0.1",
+    "ts-loader": "^7.0.2",
+    "typescript": "^3.8.3",
+    "webpack": "^4.43.0",
+    "webpack-dev-server": "^3.10.3",
+    "webpack-merge": "^4.2.2"
+  },
+  "scripts": {
```

Comment:
nit: but, I prefer to see the dependencies near the bottom, since more often newbies want to see the authorship details and scripts when onboarding or researching.

Reply:
If this is something we feel is important to standardize on, can we create an issue to discuss it? My fear is that we'll end up in a long discussion on opinions with no action without a formal resolution.